src/main/groovy/wooga/gradle/snyk/SnykPlugin.groovy

```diff
@@ -55,9 +56,18 @@ class SnykPlugin implements Plugin<Project> {
             mapExtensionPropertiesToMonitorTask(it, extension)
         }
 
+        tasks.withType(SnykInstall).configureEach {installTask ->
+            installTask.installationDir.convention(extension.snykPath)
+            installTask.executableName.convention(extension.executable)
+            installTask.snykVersion.convention(extension.snykVersion)
+        }
+        def snykInstall = tasks.register("snykInstall", SnykInstall)
+
         tasks.withType(SnykTask).configureEach {
+            dependsOn(snykInstall)
             it.executable.convention(extension.executable)
             it.snykPath.convention(extension.snykPath)
+            it.snykVersion.convention(extension.snykVersion)
```

Comment:
Why is the version mapped to all tasks? The tasks only care about the snykpath/executable.

Reply:
I have no idea why, indeed. Removed.